presto-tests/src/test/resources/queue_config_dashboard.json

```diff
@@ -17,8 +17,8 @@
     {
       "source": "(?i).*dashboard.*",
       "queues": [
-        "user.${USER}",
         "dashboard.${USER}",
+        "user.${USER}",
```

Comment:
Actually, no. The first will run, the second will queue, and the third will be rejected. The user will then be allowed to run 2 more adhoc queries and queue up to 2 more adhoc queries